opensrp-opd/src/main/java/org/smartregister/opd/activity/BaseOpdFormActivity.java

```diff
@@ -140,13 +140,13 @@ public void onClick(DialogInterface dialog, int which) {
         }
     }
 
-    private void clearSavedSession() {
+    protected void clearSavedSession() {
```

Comment:
Was this for testing only? If so we can add annotation _@VisibleForTesting_ here

Reply:
done